### PR TITLE
upgrade packages and implementation to v2

### PR DESCRIPTION
**Pull request by joecohens**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
+import Router from 'next/router'
```
Comment (sergiodxa) [inline src/index.js]: We should receive Router as argument on line 5 and then pass it too withGa and withFBQ.

When trying to import next/router directly on the dependency caused Router to be undefined always.

Comment (joecohens) [reply src/index.js]: Completley agreed! I've updated the PR passing the router as a param instead of a dependency.

**Comment by sergiodxa** [comment]
Great, will publish under v2 today

**Comment by joecohens** [comment]
🍻

**Pull request merged**